src/contents/all/components/app/LoginForm.tsx

```diff
@@ -0,0 +1,101 @@
+/*global chrome*/
+
+import React, { useEffect, useState } from 'react';
+import Frame, { FrameContextConsumer } from 'react-frame-component';
+import { useDispatch, useSelector } from "react-redux";
+import { Link } from 'react-router-dom';
+import { hot } from 'react-hot-loader/root';
+import * as Sentry from '@sentry/browser';
+
+import { Form, Icon, Input, Button, message, notification } from 'antd';
+
+import './App.scss';
+
+//import './LoginForm.scss'
+
+interface Props {
+  form: any
+}
+
+const FormItem = Form.Item;
+
+const LoginForm = () => {
```

Comment:
mainframe Login has {form, history} as props... you mentioned history isn't necessary but should i still be passing a form object?

Reply:
So this `form` prop comes from ant-design's higher order component `Form.create`, which you wrap around LoginForm like so: `Form.create<Props>()(Login)`